### ISSUE TITLE
Rename Sender/Receiver

### ISSUE THREAD

**Issue by defi-er** [kleros/kleros#164]
The current reference titles for case participants is sender and receiver. This could be deemed confusing for jurors. It could also cause the case to linger on if a wrong decision is made, in which a juror would lose a stake.

**Comment by satello** [comment]
What application is this referring to?

**Comment by defi-er** [comment]
This is referring to how parties in court cases appear for jurors. Chosen jurors have voiced opinions in the chat about how this is confusing and would like better titles.

**Comment by greenlucid** [comment]
> This is referring to how parties in court cases appear for jurors. Chosen jurors have voiced opinions in the chat about how this is confusing and would like better titles.

Your issue is still worded strangely, you might be in the wrong place, you are the one being confusing! This repo is for Kleros' smart contracts, they are in Solidity, and the naming "Sender/Reciever" the parties see in apps can't come from here. I suspect you refer to how parties are referred to in [court.kleros.io](url)? In that case, you should [make this issue in here](https://github.com/kleros/court/issues).

**Comment by epiqueras** [comment]
I think they are referring to Escrow cases?

**Comment by ferittuncer** [comment]
> > This is referring to how parties in court cases appear for jurors. Chosen jurors have voiced opinions in the chat about how this is confusing and would like better titles.
> 
> Your issue is still worded strangely, you might be in the wrong place, you are the one being confusing! This repo is for Kleros' smart contracts, they are in Solidity, and the naming "Sender/Reciever" the parties see in apps can't come from here. I suspect you refer to how parties are referred to in [court.kleros.io](url)? In that case, you should [make this issue in here](https://github.com/kleros/court/issues).

You are right but actually, Court is not responsible of this. Each arbitrable application sets its own aliases for addresses. See https://github.com/ethereum/EIPs/issues/1497 section "aliases".
